Dockerfile

```diff
@@ -52,4 +52,4 @@ COPY --chown=1001:0 --from=build /opt/app-root/src/.bundle /opt/app-root/src/.bu
 
 ENV RAILS_ENV=production RAILS_LOG_TO_STDOUT=true HOME=/opt/app-root/src DEV_DEPS=$devDeps
 
-CMD ["/opt/app-root/src/entrypoint.sh"]
+CMD ["/opt/app-root/src/scripts/entrypoint.sh"]
```

Comment:
Doesn't `scripts/` folder contain process scripts as well? Entrypoint is part of the app, and should be always accessible within the bundle/container. I think that it is more common to have entrypoint at the app's root.

Reply:
> Doesn't scripts/ folder contain process scripts as well?
I'm sorry I don't understand what you mean with "process scripts" - This PR tried to put all the different Bash script files this repository maintains under one directory. That said, it is unfortunate that I didn't find a good way to remove `pr_check.sh` and `build_deploy.sh` from the root directory. 

> Entrypoint is part of the app, and should be always accessible within the bundle/container
I don't see how that is not achievable on the Dockerfile where we can configure the image as we want and put files where we feel they should be. However I would argue there's little value in adding an extra step while building the image when it's only a route we configure. 

Arranging the scripts under one directory is just me trying to do some housekeeping, we could decide where we want to put the files depending on their nature / role. But I'd like not to have them scattered around like we have them today. I'm very open to suggestions on how we should organize them, or to be referred to some best practices if anyone can share.

Reply:
By "process" I've meant scripts that handle everything around the app, like build, ci, dev setup, support... `entrypoint` is not part of that. I agree that putting all scripts together on one place is a good idea. However, I would not want the entrypoint being mixed up with support scripts. Entrypoint is part of the app, support scripts usually are not, hence my suggestion to keep it on the root.

Reply:
@vkrizan  okay, fair enough, thank you for clarifying, `entrypoint.sh` may make sense to remain in the root dir. I believe the same applies to `devel.entrypoint.sh`  ?

Reply:
I'd guess so, yes.

Reply:
For these `pr_check.sh` and `build_deploy.sh` I guess the build options for Jenkins templates can be used, like `build_deploy_script_path` but I'm not fully sure if that would work.

Reply:
@vkrizan  you're right, but , that comes with creating a custom template for Compliance, and I don't think that the trade-off of maintaining a custom JJB template for us alone is worth it (we did remove the one we had for `codecov` token for this reason)

Reply:
@vkrizan  since `entrypoint.sh` , `build_deploy.sh` and `pr_check.sh`  are left in the root dir, can be say it's good to resolve this ?